lib/Mojolicious/Plugin/AssetPack/Pipe/OptiPng.pm

```diff
@@ -0,0 +1,94 @@
+
+package Mojolicious::Plugin::AssetPack::Pipe::OptiPng;
```

Comment:
Well, there are other PNG optimization programs, too, like https://pngquant.org/ - but people might not necessarily be interested in running all of them (pngquant for example performs lossy optimization). Plus configuring pipes isn´t particularly elegant right now, so I thought I´d keep these things apart and easy to assemble by confining the pipe to a single tool.